lib/managers/KalturaTrackingManager.js

```diff
@@ -0,0 +1,145 @@
+const util = require('util');
+const udpsender = require('../utils/udpsender');
+const crypto = require('crypto');
+const kaltura = require('../KalturaManager');
+
+/**
+ * @service tracking
+ */
+class KalturaTrackingManager extends kaltura.KalturaManager
+{
+    constructor()
+    {
+        super();

```

Comment:
if the constructor - like here is empty - if you just remove it , it will call super as default 
So I would remove the constructor here entirely